packages/components/src/lozenge/src/Lozenge.tsx

```diff
@@ -45,6 +45,10 @@ const textSize = createSizeAdapter({
 });
 /* eslint-enable sort-keys, sort-keys-fix/sort-keys-fix */
 
+const textTransform = function (size) {
+    return size === "sm" ? "uppercase" : null;
```

Comment:
Not familiar with this but does that mean that we would set `textTransform: null`? Do we want it to be `textTransform: undefined`?